dev/src/sdlsystem.cpp

```diff
@@ -114,6 +114,20 @@ void updatemousebutton(int button, int finger, bool on) {
     updatebutton(name, on, finger);
 }
 
+void updatepressedkeys(string kname, Uint8 state)
+{
+    switch (state) {
+        case SDL_RELEASED:
+            pressedkeys.erase(kname);
+            break;
+        case SDL_PRESSED:
+            pressedkeys.insert(kname);
```

Comment:
this also doesn't seem usual, this way you only know if the key is down, not if it went down